controllers/serviceexport/controller.go

```diff
@@ -0,0 +1,143 @@
+package serviceexport
+
+import (
+	"time"
+
+	apicorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/informers"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	discoveryinformers "k8s.io/client-go/informers/discovery/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/workqueue"
+)
+
+// TO-DO (chenyu1): Update the API information when the design is finalized
+const (
+	// ServiceExport GVR information
+	svcExportGroup    = "mcs.networking.fleet.azure.io"
+	svcExportVersion  = "v1alpha1"
```

Comment:
I put them there because at this moment we do not have an `api` package yet (which should probably be sent as a separate PR when the design is finalized); I am not 100% sure about putting these information in `groupversion_info.go` is a good idea considering it is all generated code; I will replace them with `SchemeGroupVersion.WithResource()` calls when the package becomes available.

Reply:
ack, could you please add a comment before this section to indicate that?

Reply:
A TO-DO comment has been added 👌